plugins/bucketize_procedure.cc

```diff
@@ -224,7 +228,9 @@ run(const ProcedureRunConfig & run,
         DEBUG_MSG(logger) << "Bucket " << mappedRange.first << " from "
                           << lowerBound << " to " << higherBound;
 
-        parallelMap(lowerBound, higherBound, applyFct);
+        if (!parallelMapHaltable(lowerBound, higherBound, applyFct)) {
+            DEBUG_MSG(logger) << BucketizeProcedureConfig::name << " procedure was cancelled";
```

Comment:
Procedure was cancelled notification should be handled by the caller, not in each procedure.